MAUI/TabView/Header-Display-Mode.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Header display mode in .NET MAUI Tab View control | Syncfusion
+description: Learn here all about header display mode support in Syncfusion .NET MAUI Tab View (SfTabView) control and more.
+platform: maui
+control: TabView
+documentation: ug
+---
+
+# Header display mode in .NET MAUI Tab View control (SfTabView)
+
+By default, the Tab View control displays the title of each Tab Item. It can be changed to any of the following types:
```

Comment:
By default, the .NET MAUI Tab View control displays the title of each Tab Item. It can be changed to any of the following types:

Reply:
Changed and added API link